compiler/testData/codegen/bytecodeText/deadCodeElimination/simpleConstructorNotRedundant.kt

```diff
@@ -9,12 +8,19 @@ fun box() {
     else {
         y = null
     }
-    
+
     y!!
 }
 
+// JVM_TEMPLATES:
 // 0 IFNULL
 // 1 IFNONNULL
 // 1 throwNpe
 // 0 ATHROW
 // 0 checkNotNull
+// JVM_IR_TEMPLATES:
+// 0 IFNULL
+// 0 IFNONNULL
+// 0 throwNpe
```

Comment:
This should probably be `// 1 throwNpe`

Reply:
Also note that you can extract common parts for additional clarity:
```
// 0 IFNULL
// 1 throwNpe
// 0 ATHROW
// 0 checkNotNull

// JVM_TEMPLATES:
// 1 IFNONNULL
// JVM_IR_TEMPLATES:
// 0 IFNONNULL
```

Reply:
Right, fixed and refactored!